ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/tests/ContractKeysNo1866.scala

```diff
@@ -0,0 +1,176 @@
+// Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.api.testtool.tests
+
+import java.util.UUID
+
+import com.daml.ledger.api.testtool.infrastructure.{LedgerSession, LedgerTest, LedgerTestSuite}
+import com.digitalasset.ledger.test_1_6.DA.Types.Tuple2
+import com.digitalasset.ledger.test_1_6.Test.Delegation._
+import com.digitalasset.ledger.test_1_6.Test.ShowDelegated._
+import com.digitalasset.ledger.test_1_6.Test.TextKey._
+import com.digitalasset.ledger.test_1_6.Test.TextKeyOperations._
+import com.digitalasset.ledger.test_1_6.Test._
+import io.grpc.Status
+
+final class ContractKeysNo1866(session: LedgerSession) extends LedgerTestSuite(session) {
```

Comment:
As mentioned offline, let's rename this simply to `ContractKeys` as it's the main test for the latest active stable version of DAML-LF.